### PR TITLE
chore: revamp readme

### PR DESCRIPTION
**Pull request by NabilSli**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -759,8 +758,8 @@ footer {
     }
 
     .icon-info {
-        width: 31px;
-        height: 23px;
+        width: 37px;
+        height: 27px;
```
Comment (tony-go) [inline css/style.css:762]: Not sure it's related to the readme tho.

**Review by tony-go — CHANGES_REQUESTED** [review]
You should delete images there ^^

File: README.md [hunk 2]
```diff
@@ -1,41 +1,40 @@
-# BOOKI: Exemple of a travel agency website.
-PROJECT 2 of OpenClassroom's Web developer Course
+# 😃 BOOKI: Exemple of a travel agency website.
+This website is my first achievement as a web developer👏. I did it during my boot camp at OpenClassroom's Web developer.
 
-## Use the full range of knowledge acquired during the courses.
+## 👀 Here you can check the final result
+https://645a7889605f5a058dad93d1--incredible-shortbread-990e7e.netlify.app/#
```
Comment (tony-go) [inline README.md:5]: Use an anchor please here 
```suggestion
[See the web site](https://645a7889605f5a058dad93d1--incredible-shortbread-990e7e.netlify.app/#)
```

File: README.md [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+
+## ⭐️ Use the full range of knowledge acquired during the courses.
 To realise this project there was some technical skill required, such as:
 
 
-* html
+* 🎯 `HTML`
   * Build the page using the right tabs
```
Comment (tony-go) [inline README.md]: What the `right tabs` means ?

File: README.md [hunk 4]
```diff
@@ -1,41 +1,40 @@
...
   * dividing the page in blocks from the wireframe to ensure a responsive design
```
Comment (tony-go) [inline README.md]:
```suggestion
  * Dividing the page into blocks from the wireframe to ensure a responsive design
```

File: README.md [hunk 5]
```diff
@@ -1,41 +1,40 @@
...
   * Search engine optimization
```
Comment (tony-go) [inline README.md]: Precise which one?

File: README.md [hunk 6]
```diff
@@ -1,41 +1,40 @@
...
   * Good syntaxe, comments and indent to make the code easily maintened and readable
   * Use css properties in order to not repeat the same code multiple time
```
Comment (tony-go) [inline README.md]: Should be in the CSS part ?

File: README.md [hunk 7]
```diff
@@ -1,41 +1,40 @@
...
                                                                            
-* Css
+* 📚 `CSS`
   * Hierachise the code
   * No repetion
```
Comment (tony-go) [inline README.md]:
```suggestion
  * Use CSS properties in order to not repeat the same code multiple time
```

File: README.md [hunk 8]
```diff
@@ -1,41 +1,40 @@
...
+* 🎯 `HTML`
   * Build the page using the right tabs
   * dividing the page in blocks from the wireframe to ensure a responsive design
   * Search engine optimization
   * Good syntaxe, comments and indent to make the code easily maintened and readable
   * Use css properties in order to not repeat the same code multiple time
                                                                            
-* Css
+* 📚 `CSS`
   * Hierachise the code
```
Comment (tony-go) [inline README.md]: Add more detail; which pattern did you follow?

File: README.md [hunk 9]
```diff
@@ -1,41 +1,40 @@
...
   * No repetion
   * Use media query's to ensure reponsive design
   * Desktop first focus for this project
   * Use flexbox and/or grid
 
-* Git and GitHub
+* 👨‍👩‍👧‍👧 `GIT`
   * Use git for vesrioning
   * Make pullrequest and have reviews of your work
   * resolve issues
```
Comment (tony-go) [inline README.md]:
```suggestion
  * Use `git` for versioning
  * Create branches and open pull requests for each new feature
  * Get reviewed and apply proper corrections
```

File: README.md (hunk 3 above)
Comment (NabilSli) [reply README.md]: i meant tags

File: README.md (hunk 2 above)
Comment (NabilSli) [reply README.md:5]: done :D

File: README.md (hunk 3 above)
Comment (NabilSli) [reply README.md]: 🤣

File: README.md (hunk 6 above)
Comment (NabilSli) [reply README.md]: i was talking about css class in the html code

File: README.md [hunk 10]
```diff
@@ -1,31 +1,30 @@
...
 
 ## ⭐️ Use the full range of knowledge acquired during the courses.
 To realise this project there was some technical skill required, such as:
 
 
 * 🎯 `HTML`
-  * Build the page using the right tabs
-  * dividing the page in blocks from the wireframe to ensure a responsive design
+  * Build the page using the right tags
+  * dividing the page into blocks from the wireframe to ensure a responsive design
```
Comment (tony-go) [inline README.md:13]:
```suggestion
  * Dividing the page into blocks from the wireframe to ensure a responsive design
```

File: README.md [hunk 11]
```diff
@@ -1,31 +1,30 @@
...
   * Search engine optimization
-  * Good syntaxe, comments and indent to make the code easily maintened and readable
-  * Use css properties in order to not repeat the same code multiple time
+  * code quality practices: comments and indent to make the code easily maintained and readable
+  * Use the right css class in order to not repeat the same code multiple time
```
Comment (tony-go) [inline README.md:16]:
```suggestion
  * Factorize CSS class in order not to repeat the same code multiple time
```

File: README.md [hunk 12]
```diff
@@ -1,31 +1,30 @@
...
                                                                            
 * 📚 `CSS`
-  * Hierachise the code
+  * Hierachise the code, starting witrh the desktop view as we are in a desktop first project, going top to bottom and from the bigger to the smaller view
```
Comment (tony-go) [inline README.md:19]:
```suggestion
  * Hierachise the code, starting with the desktop view as we are in a desktop-first project, going top to bottom and from the bigger to the smaller view
```

File: README.md (hunk 7 above)
Comment (tony-go) [reply README.md:20]: You forgot this ^^

**Review by tony-go — COMMENTED** [review]
We get closer 💪🏼

**Review by tony-go — APPROVED** [review]
🎉

**Pull request merged**